### PR TITLE
Automatic translation build for build travis 

### PR DESCRIPTION
**Pull request by stephdl**
NethServer/dev#6629

### CODE REVIEW

File: .transifex.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# set python3 and upgrade pip (default pip 10.0)
+pyenv global 3.7.1
+pip3 install --upgrade pip
+
+pip install virtualenv
+virtualenv ~/env
+source ~/env/bin/activate
+pip install transifex-client
+echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"api"$'\npassword = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc
+tx push -s
```
Comment (DavidePrincipi) [inline .transifex.sh:30]: With Python 3 pyenv/virtualenv shouldn't be necessary:
```suggestion
python3 -mvenv ./venv --upgrade-deps
./venv/bin/pip3 install transifex-client
echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"api"$'\npassword = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc
./venv/bin/tx push -s
```

Comment (stephdl) [reply .transifex.sh:30]: sorry to bother you @DavidePrincipi but the command above fails, this is the error log traces

```
0.10s$ ./.transifex.sh
We push the translations to transifex
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip]
            ENV_DIR [ENV_DIR ...]
venv: error: unrecognized arguments: --upgrade-deps
./.transifex.sh: line 22: ./venv/bin/pip3: No such file or directory
./.transifex.sh: line 24: ./venv/bin/tx: No such file or directory
```

this is the commit I did https://github.com/NethServer/nethserver-sogo/pull/53

Comment (DavidePrincipi) [reply .transifex.sh:30]: Ok, :white_flag:

**Review by DavidePrincipi — APPROVED** [review]

**Review by gsanchietti — APPROVED** [review]
Sorry, I forgot the review request

**Pull request merged**